ui/app/src/state/reducers/preview.ts

```diff
@@ -570,6 +570,19 @@ const reducer = createReducer<GlobalState['preview']>(
       ...state,
       previewChoice: { ...state.previewChoice, ...JSON.parse(payload.previewChoice ?? '{}') }
     }),
+    [changeSite.type]: (state) => {
+      return {
+        ...state,
+        toolsPanelPageStack: [],
+        toolsPanel: null,
+        pageBuilderPanel: null,
+        toolbar: {
+          leftSection: null,
+          middleSection: null,
+          rightSection: null
+        }
+      };
+    },
```

Comment:
I think a more comprehensive reset is in order here.

Will merge this to kill the bug but please look into a more complete reset (understanding each property and acting accordingly).